Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml

```diff
@@ -923,6 +925,12 @@
     sprite: Clothing/Uniforms/Jumpsuit/ert_leader.rsi
   - type: Clothing
     sprite: Clothing/Uniforms/Jumpsuit/ert_leader.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
```

Comment:
okay when i said a lil bit i meant like 0.3-0.5 max

Reply:
Wouldn't that be more protection than they already have?

Reply:
This is the protection they currently provide with the values 0.9 and 0.8 respectfully.

![image](https://user-images.githubusercontent.com/110078045/212793989-3552451a-e858-4f55-a441-61e6f8528b54.png)
![image](https://user-images.githubusercontent.com/110078045/212794014-63c21491-8e42-4263-a229-828cd0b85bb0.png)

Reply:
huh, i dont remember it working like that, this is fine thenb